Zscaler/zscaler-zia/tests/test_event_audit1.json

```
@@ -0,0 +1,34 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Zscaler Internet Access [BETA]",
+        "dialect_uuid": "dc0f339f-5dbe-4e68-9fa0-c63661820941"
+      }
+    },
+    "message": "{\"sourcetype\": \"zscalernss-audit\",\"event\": {\"time\": \"Mon Aug 28 08:04:30 2023\",\"recordid\": \"0\",\"action\": \"UPDATE\",\"category\": \"Unknown\",\"subcategory\": \"Unknown\",\"resource\": \"com.zscaler.zapi.domain.nss.ZmanageNssFeed\",\"interface\": \"UI\",\"adminid\": \"None\",\"clientip\": \"1.2.3.4\",\"result\": \"SUCCESS\",\"errorcode\": \"None\",\"auditlogtype\": \"ZIA\",\"preaction\": \"Unknown\",\"postaction\": \"Unknown\"}}"
+  },
+  "expected": {
+    "message": "{\"sourcetype\": \"zscalernss-audit\",\"event\": {\"time\": \"Mon Aug 28 08:04:30 2023\",\"recordid\": \"0\",\"action\": \"UPDATE\",\"category\": \"Unknown\",\"subcategory\": \"Unknown\",\"resource\": \"com.zscaler.zapi.domain.nss.ZmanageNssFeed\",\"interface\": \"UI\",\"adminid\": \"None\",\"clientip\": \"1.2.3.4\",\"result\": \"SUCCESS\",\"errorcode\": \"None\",\"auditlogtype\": \"ZIA\",\"preaction\": \"Unknown\",\"postaction\": \"Unknown\"}}",
+    "event": {
+      "kind": "event",
+      "action": "update",
+      "category": ["process"],
+      "type": ["info"]
```

This event match a change in the configuration of the tenant. Can you update the parser to set `event.category` and `event.type` accordingly?
```suggestion
      "category": ["configuration"],
      "type": ["change"]
```